src/Storages/LiveView/StorageLiveView.cpp

```diff
@@ -239,7 +239,6 @@ void StorageLiveView::writeIntoLiveView(
     copyData(*data, *output);
```

Comment:
Asan says that context is not alive here, but I can't understand why.

Reply:
Ok, I will take a look.